packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -123,6 +132,11 @@ export interface CommonFormFieldProps {
    * set it to `always`.
    */
   readonly clearable?: Clearable;
+
+  /**
+   * Which version of the Field to use
+   */
+  version?: 1;
```

Comment:
Also... I think it might be good to include a bit more description of what this is for, and that setting it to anything but 1 at this point is experimental/unsupported.

Reply:
I added a comment but the type being restrictive makes sense. If we do another version in the future we could have it be `version ?: 1 | 2 |3` or something like that
[6d4ace1](https://github.com/GetJobber/atlantis/pull/2184/commits/6d4ace130120a195fd0c8160b1f1209c89982dff)